docs/customization/newsfile.rst

```diff
@@ -25,3 +25,10 @@ In an existing news file, it'll look something like this::
   - Fixed, etc...
 
 ``towncrier`` will not alter content above the comment.
+
+Markdown
+~~~~~~~~
+
+If your news file is in Markdown (e.g. ``NEWS.md``), use the following comment instead::
+
+  <!--- towncrier release notes start -->
```

Comment:
If you don't explicitly add a type, it's interpreted as Python, but it's HTML: https://towncrier--483.org.readthedocs.build/en/483/customization/newsfile.html

`.. code-block:: html`

Bonus points for adding the other code blocks to `rat`. 🤓